tests/macos/xcodeproj/Single-Application-Project-AllTargets.xcodeproj/bazelinstallers/indexstores.sh

```diff
@@ -4,11 +4,23 @@ set -euo pipefail
 
 # See `_indexstore.sh` for full details.
 
-# Make sure add these to bazel build target copts for objective-c
-find "$BAZEL_WORKSPACE_ROOT/bazel-out"/*/bin/ \
-     -type d \
-     -name "*.indexstore" \
-     -print0 \
-    | xargs -0 "$BAZEL_INSTALLERS_DIR/_indexstore.sh"
-
-echo "Finish remapping index files"
\ No newline at end of file
+echo "Start remapping index files at `date`"
+
+FOUND_INDEXSTORES=`pcregrep -o1 'command_line: "(.*\.indexstore)' $bazel_build_event_text_filename || true`
+declare -a EXISTING_INDEXSTORES=()
+for i in $FOUND_INDEXSTORES
+do
+if [ -d $i ]
+then
+EXISTING_INDEXSTORES+=($i)
```

Comment:
For simpler projects (like the ones in this repo), no unit files and no indexstore directory will be generated. In that case, we shouldn't try to move non-existent directories!

Reply:
> For simpler projects (like the ones in this repo), no unit files and no indexstore directory will be generated

If that's the case, can you please add a new project that will exercise this code path?

Reply:
I realize now that the unit files were not being generated because sandboxing was enabled. Bazel would build the outputs and then delete any other files left behind (including the generated indexstores.)

For now, if anyone would like to manually test index generation in the rules_ios repo, just make sure to disable sandboxing by adding these lines to .bazelrc:

```
build --genrule_strategy=standalone
build --spawn_strategy=standalone
```